Tarea/.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
+
```

Comment:
You only need to add html, css and js, we don't need this file

Reply:
This item was deleted.

PD: I modified my task, I just made a new commit with a basic js.